app/views/users/profile_edit/_donation_bar.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="donation-bar-container" data-cause="<%= cause %>" data-percent="<%= percent %>" data-is-highest="<%= highest %>">
```

Comment:
I don't think you have to end this element with `-container`, and rename to just `donation-bar`, as `container` classes in css is more used as a "general" purpose to refer to reusable CSS classes

Reply:
I use container because it contains the bar, the cause text and the percentage. Maybe I can rename the partial to `donation_bar_container`. What do you think?
There's actually a possibility of this container being re-used. I was considering re-using it on the profile page instead of what I currently have there. But that's not part of the scope of this pr.

Reply:
So when I mean reusability I meant the literal class name.

`container` is a term heavily used in bootstrap in where if you apply the class `container` in other elements, it applies basic margin, padding and maybe few other things to get the grid going. 

You will probably not going to apply `donation-bar-container` class to other elements. If you're talking about this whole `erb` layout, sure we can use that multiple times, but not the actual class names. We're probably not going to be in a word where we have
```
<div class="profile donation-bar-container">
```

Reply:
alright, cool. I can change it.